atat/domain/csp/cloud/models.py

```diff
@@ -40,10 +59,19 @@ class AliasModel(BaseModel):
     * user_object_id:objectId
     """
 
+    reset_stage: bool = False
+
     class Config:
         alias_generator = snake_to_camel
         allow_population_by_field_name = True
 
+    def dict(self, *args, **kwargs):
+        kwargs.setdefault("exclude")
+        if kwargs["exclude"] is None:
+            kwargs["exclude"] = set()
```

Comment:
I think this would give you the same result:
```suggestion
        kwargs.setdefault("exclude", default=set())
```

Reply:
I thought about this too, but I found that in this instance, `exclude=None` is mysteriously being passed in as a kwarg, so even if I supplied a default with `setdefault`, I had to check for `None` anyway.